pallets/controller/rpc/runtime-api/src/lib.rs

```diff
@@ -53,6 +53,20 @@ pub struct BalanceInfo {
 	pub amount: Balance,
 }
 
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Eq, PartialEq, Encode, Decode, Default, RuntimeDebug)]
+pub struct BalanceInfoExtended {
+	#[cfg_attr(feature = "std", serde(serialize_with = "serialize_as_string"))]
+	#[cfg_attr(feature = "std", serde(deserialize_with = "deserialize_from_string"))]
+	pub pool_total_supply: Balance,
```

Comment:
```suggestion
	pub pools_total_supply_usd: Balance,
```
or
```suggestion
	pub pools_total_supply_in_usd: Balance,
```
Same for `pool_total_borrow`